src/app/application.h

```diff
@@ -103,6 +106,10 @@ private slots:
     QPointer<WebUI> m_webui;
 #endif
 
+    // FileLog
+    bool m_fileLogEnabled;
+    QPointer<FileLogger> m_fileLogger;
```

Comment:
Why not  `QScoopedPointer`? The `Application` object owns the `m_fileLogger` object, doesn't it?